src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/RoboticsCallbackHandler.java

```diff
@@ -79,6 +84,20 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             log.info("Is case valid to send to robotics {} for case id {}", isCaseValidToSendToRobotics, callback.getCaseDetails().getId());
 
             if (isCaseValidToSendToRobotics) {
+                if(gapsSwitchOverFeature){
+                    String region = latestCase.getData().getRegion();
+
+                    Map<String, RegionalProcessingCenter> regionalProcessingCenterMap = regionalProcessingCenterService
```

Comment:
As this is being repeated we should just add this method into regionalProcessingCenterService in common so it can be used here and in Tribunals